tictactoe.js

```diff
@@ -0,0 +1,139 @@
+var count = 0;
+
+var box1 = document.querySelector("#tile1");
+var box2 = document.querySelector("#tile2");
+var box3 = document.querySelector("#tile3");
+var box4 = document.querySelector("#tile4");
+var box5 = document.querySelector("#tile5");
+var box6 = document.querySelector("#tile6");
+var box7 = document.querySelector("#tile7");
+var box8 = document.querySelector("#tile8");
+var box9 = document.querySelector("#tile9");
+
+function click(box) {
+	box.addEventListener("click", function() {
+		if (box.innerHTML === "") {
+			
+			if (count % 2 ===0) {
+
+				box.style.background = "yellow";
+				box.style.color = "green";
+				box.innerHTML = "X";
+				count++;
+			} else {
+				box.style.background = "green";
+				box.style.color = "yellow";
+				box.innerHTML = "O";
+				count++;
+			}
+		}
+	})
+}
+
+click(box1);
+click(box2);
+click(box3);
+click(box4);
+click(box5);
+click(box6);
+click(box7);
+click(box8);
+click(box9);
```

Comment:
Another good option would be to select the boxes by class name. Here's an example:

```js
var boxes = document.getElementsByClassName("tile");
```

Then you could call these functions by going through a for loop.